pkg/dump/writer/tar_writer_test.go

```diff
@@ -71,7 +71,7 @@ func TestTarWriter_Write(t *testing.T) {
 	writer.Flush(ctx)
 	writer.Close(ctx)
 
-	err = puller.ExtractTarGz(writer.OutputPath(), tmpTarExtractDir, config.DefaultMaxArchiveSize)
+	err = puller.ExtractTarGz(false, writer.OutputPath(), tmpTarExtractDir, config.DefaultMaxArchiveSize)
```

Comment:
nit: maybe using a variable with a name here (and the other caller), instead of true/false? (since the signature of the function changed it's a bit confusing on what "false" means)

Reply:
changed